### DISCUSSION TITLE
關於GPL-3.0的使用問題

### DISCUSSION THREAD

**Discussion by leungyauming** [CaiJimmy/hugo-theme-stack#784] [General]
我最近在用GitHub Pages架設自己的blog網站，然後因為那個organisation不是enterprise之類的所以不可以在private repository用GitHub Pages，我只能用public repository。
然後我發現你這個專案在使用GPL-3.0，條文中有規定所有publicly available的derivative work都必須使用GPL-3.0。
雖然我對於自己的blog用GPL-3.0沒有很在意，但是我未來有可能會再用這個theme。
所以我想看一下作者你對於別人用你的theme，然後想換成更permissive或者其他license這件事情的看法。
然後LGPL-3.0會不會是一個更好的選擇呢？

**Comment by CaiJimmy** [comment]
抱歉，其实我对开源协议不是很熟悉，所以可能不能回复你的问题。（当时是随便选了一个，现在要改也很麻烦了。）

> 所以我想看一下作者你對於別人用你的theme，然後想換成更permissive或者其他license這件事情的看法。

二次修改主题的话我会希望它开源啦，如果能遵循协议当然是最好了。
